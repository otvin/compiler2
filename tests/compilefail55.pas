program compilefail55(output);
{should error when you do not assign the result of a function}
function oops(a:integer):integer;
begin
	a:=a+7;
	oops:=a*2;
end;

begin
	oops(3);
end.
