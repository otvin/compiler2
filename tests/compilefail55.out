Cannot invoke function without assigning its return value:  'oops' in file:tests/compilefail55.pas line:10    column:5  
